raft-tech/TANF-app
# Bug 2379 search form error count

## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #2379 

<img width="341" alt="Screen Shot 2023-03-09 at 12 14 59 PM" src="https://user-images.githubusercontent.com/7119690/224145412-16f1ecde-7180-4438-a970-9cc0f358580f.png">
<img width="346" alt="Screen Shot 2023-03-09 at 12 15 06 PM" src="https://user-images.githubusercontent.com/7119690/224145529-e3ef754c-b324-4f19-b7fe-df4d6d9f7fbe.png">
<img width="377" alt="Screen Shot 2023-03-09 at 12 16 30 PM" src="https://user-images.githubusercontent.com/7119690/224145623-3d4c092c-ce9c-41ad-867d-e929b09f68b8.png">
<img width="347" alt="Screen Shot 2023-03-09 at 12 16 36 PM" src="https://user-images.githubusercontent.com/7119690/224145638-e697c69c-811a-4b26-9026-ec08d3ee8113.png">
<img width="365" alt="Screen Shot 2023-03-09 at 12 16 40 PM" src="https://user-images.githubusercontent.com/7119690/224145640-5a5fe813-30ab-48f6-a2f5-a588580fa743.png">


## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/ and sign in.
2. Go to datafile tab.
3. As a Data Analyst, submit a form with unselected fields and confirm error count is as expected.
4. As an OFA Admin, submit a form with unselected fields and confirm error count is as expected.

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
